src/base/DeltaResolver.sol

```diff
@@ -82,16 +82,14 @@ abstract contract DeltaResolver is ImmutableState {
     /// @notice Calculates the amount for a take action
     function _mapTakeAmount(uint256 amount, Currency currency) internal view returns (uint256) {
         if (amount == Constants.OPEN_DELTA) {
-            return _getFullCredit(currency).toUint128();
```

Comment:
i think this was accidentally copied from the  `mapInputAmount` function which returns a uint128